c/rce_validator.c

```diff
@@ -92,27 +92,35 @@ static int make_witness_cursor(uint8_t* buffer, uint32_t cache_size,
 }
 
 int main() {
+  // Load Script args first.
+  //
+  // | Script Args   | Type ID | Flags |
+  // | ------------- |---------| ------|
+  // | Length(Byte)  | 32      | 1     |
+
+  uint8_t current_script[32768];
```

Comment:
we have such kind macro, instead of magic number.

Reply:
I can't find any macro in ckb-c-std-lib

Reply:
we can define one "#define SCRIPT_SIZE 32768" :it's frequently used.

Reply:
sure

Reply:
The script **must** be less than 32768, is this a convention?